pkgs/by-name/te/tetrio-desktop/package.nix

```diff
@@ -15,19 +15,24 @@
 , gtk3
 , libpulseaudio
 , systemd
-, callPackage
-, withTetrioPlus ? false
-, tetrio-plus ? callPackage ./tetrio-plus.nix { }
+, withTetrioPlus ? false # For backwards compatibility. At the time of writing, the latest released tetrio plus version is not compatible with tetrio desktop.
+, tetrio-plus ? throw "tetrio-plus is not supported on version 9"
```

Comment:
I actually think this is not a so good idea.

Reply:
What do you think is wrong?

Reply:
Because if it's not set, the throw will never be evaluated. And if it's set, it won't be used later on in the file. Therefore, the `throw` is totally useless there.

Reply:
Oh, I didn't see the `postInstall` that used it was removed. Should that be added back?

Reply:
No, I don't think so.

Reply:
So those params should be removed for now, and everything can be re-added once TETR.IO Plus is working?

Reply:
Yes, that's what I would do.

Reply:
Sounds good to me.

Reply:
Feel free to adjust this PR, I won't have time to do it until tonight.

Reply:
I don't get it why though? It would break the configs of everyone who has `withTetrioPlus` set. Why not leave it as a dummy parameter that does nothing for the time being? When tetrio plus is working again, they all get it back automatically.

Reply:
That's also true. If the `postInstall` is re-added, users who have it enabled will see an error that it's not currently supported, but could still provide their own nightly build of it. Silently removing the option doesn't sound so good. At the very least, if it stays removed for now, a warning should be emitted when `withTetrioPlus` is enabled.

Reply:
Ready to merge for you then?